### PR TITLE
cointoolkit - blockbook backend

### PR DESCRIPTION
**Pull request by coladonz**
#4 Implement support for blockbook backend - fixed

### CODE REVIEW

File: js/cointoolkit.js [hunk 1]
```diff
@@ -998,6 +998,139 @@ $(document).ready(function() {
...
+					},
+				});
+			}
+		},
+		broadcast: function(endpoint) {
+			return function(thisbtn){
+				var orig_html = $(thisbtn).html();
+				$(thisbtn).html('Please wait, loading... <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>').attr('disabled',true);
+				$.ajax ({
+					type: "GET",
```
Comment (backpacker69) [inline js/cointoolkit.js]: please do it as a POST request

File: js/cointoolkit.js [hunk 2]
```diff
@@ -1846,35 +1979,35 @@ var bcBasedExplorer = {
 		},
 		peercoin: {
 			listUnspent: {
-				"iquidus": iquidusBasedExplorer.listUnspent('https://explorer.peercoin.net'),
+				"iquidus": peerBlockbookBasedExplorer.listUnspent('https://blockbook.peercoin.net'),
```
Comment (backpacker69) [inline js/cointoolkit.js]: shouldn't it now read blockbook

File: js/cointoolkit.js [hunk 3]
```diff
@@ -998,6 +998,139 @@ $(document).ready(function() {
...
+							} else {
+								$("#redeemFromStatus").removeClass('hidden').html('<span class="glyphicon glyphicon-exclamation-sign"></span> Unexpected error, unable to retrieve transation info');
+								callback(false);
+							}
+						}
+//,
+//						  complete: function(data, status) {
+//							  $("#redeemFromBtn").html("Load").attr('disabled',false);
+//							  totalInputAmount();
+//						  }
```
Comment (backpacker69) [inline js/cointoolkit.js]: what is this?

**Review by backpacker69 — CHANGES_REQUESTED** [review]

File: js/cointoolkit.js (hunk 1 above)
Comment (coladonz) [reply js/cointoolkit.js]: Ok, I will change this to POST request.

File: js/cointoolkit.js [hunk 4]
```diff
@@ -1071,18 +1071,18 @@ $(document).ready(function() {
...
+							  totalInputAmount();
+						  }
 					});
 				}
 			},
 		getInputAmount: function(endpoint) {
 			return function(txid, index, callback) {
 				$.ajax ({
-					type: "GET",
+					type: "POST",
```
Comment (backpacker69) [inline js/cointoolkit.js]: does /api/tx/ support post requests?

File: js/cointoolkit.js [hunk 5]
```diff
@@ -1104,7 +1104,7 @@ $(document).ready(function() {
 				var orig_html = $(thisbtn).html();
 				$(thisbtn).html('Please wait, loading... <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>').attr('disabled',true);
 				$.ajax ({
-					type: "GET",
+					type: "POST",
 					url: "" + endpoint + "/api/sendtx/"+$("#rawTransaction").val(),
```
Comment (backpacker69) [inline js/cointoolkit.js]: post does not work like that

**Review by backpacker69 — APPROVED** [review]

**Pull request merged**